packages/web/src/components/results/TeeResults.vue

```diff
@@ -218,10 +222,10 @@ const reFilteredPrograms = computed(() => {
       let bool = false
       if (filterVal === '') {
         bool = true
-      } else if (trueIf === '==') {
+      } else if (trueIf === ConditionOperators.is) {
         // console.log('TeeResults > reFilteredPrograms > progVal :', progVal )
         bool = progVal === filterVal
-      } else if (trueIf === 'exists') {
+      } else if (trueIf === ConditionOperators.exist) {
```

Comment:
suggestion: le principe des noms d'enum de `ConditionOperators` est le verbe conjugué (`.is`) à la troisième personne du singulier, et non le verbe à l'infinitif (~`.be`~) afin de définir la partie gauche de la condition comme si on lisait une phrase (`it is ...`). Le verbe "exist" conjugué ainsi donne `exists` et non `exist`

Reply:
Ok pas de souc pour ajouter le 's' ;-)

Reply:
Au passage, je rappel: #283

Et le coté verbe est pas tout à fait vrai ! On trouve dans l'existant:

```ts
  ...
  notEqual = '!=',
  superior = '>',
  superiorOrEqual = '>=',
  inferior = '<',
  inferiorOrEqual = '<=',
```

Alors, peut être que pour `is` on aurait pu avoir `equal` (?)